packages/odyssey-react-mui/src/DataTable/DataTable.tsx

```diff
@@ -201,6 +205,10 @@ export type DataTableProps = {
    * The component to display when the query returns no results
    */
   noResultsPlaceholder?: ReactNode;
+  /**
+   * An optional set of filters to render in the filters menu
+   */
+  filters?: Array<DataFilter | DataTableColumn<DataTableRowData> | string>;
```

Comment:
Why can this be a string?

Reply:
The string is so that consumers can pass in the id of an existing column

Reply:
But only one string?